pkg/nuclide-clang-rpc/spec/ClangFlagsManager-spec.js

```diff
@@ -233,6 +243,37 @@ describe('ClangFlagsManager', () => {
     });
   });
 
+  it('gets project flags in addition to compilation database flags', () => {
+    waitsForPromise(async () => {
+      const sampleJsonFile = nuclideUri.join(__dirname, 'fixtures',
+        'nuclide_clang_flags.json.sample');
+      const tmpJsonFile = nuclideUri.join(__dirname, 'fixtures', 'nuclide_clang_flags.json');
```

Comment:
Let's use a real temporary directory for this test. We have a `copyFixture` utility (see https://github.com/facebook/nuclide/blob/648d770f3f58794dbdcfa1b0d9ccf6ae484e1581/spec/clang-refactor-integration-spec.js#L25), so let's copy out the fixtures directory. The temporary directory will be automatically cleaned up so no need for the manual `afterEach` either :)